### DISCUSSION TITLE
Enhancement : Add Earn Apps to gallery

### DISCUSSION THREAD

**Discussion by ddelpire** [IceWhaleTech/CasaOS#184] [Ideas]
#### Is your idea related to a problem? Please describe:
I will be interesting to add docker adds like EarnApp, HoneyGain, IPRoyal to gallery apps

#### Describe the solution you'd like:
For the moment, i install theses apps in command line with docker-compose because these earning apps are not present in CasaOS gallery

#### Describe alternatives you've considered:

Add a category Earning apps and a wiki for installing quickly these apps (some apps don't works under ARM64 but with binfmt quemu emulator it is possible ;-) )

#### Additional context
I own an OrangePi 3 LTS  with theses apps and CasOS and it's a pity that it is not possible to integrate them into CasaOS interface

**Comment by CollabVMA50** [comment]
use honeygain's own docker version
